kernel-modules/MODULE_VERSION

```diff
@@ -1 +1 @@
-2.6.0
+2.7.0-rc1
```

Comment:
Since we are pretty close to the release, we probably want to go directly to `2.7.0`